### DISCUSSION TITLE
comandos do LCD TFT35 E3 V3.0 invertidos

### DISCUSSION THREAD

**Discussion by MarcioMergulhao** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2344] [Q&A]
olá amigos,, alguém da comunidade poderia me ajudar, estou tendo problemas com o LCD TFT35 E3 V3.0 atualizei ele e agora os comandos estão invertidos,, por exemplo, botões de baixo comandando os de cima no LCD e diz que não há impressora conectada,, será que alguém já teve esse problema e pode nos ajudar com essa questão, nós da comunidade ficaríamos muito gratos com qualquer ajuda pois sou novato nesse ramo de impressão 3D, por favor nos ajudem

**Comment by oldman4U** [comment]
Please communicate in english. Thank you
